File: CS/WindowsApplication1/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraGrid.Views.Card;

namespace WindowsApplication1
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
            new DevExpress.XtraGrid.Design.XViewsPrinting(gridControl1);
        }

        private void cardView1_KeyDown(object sender, KeyEventArgs e)
        {
            CardView Cards = sender as CardView;
            if (e.KeyCode == Keys.Insert && !Cards.IsEditing)
            {
                DataView Data = Cards.DataSource as DataView;
                if (Data != null)
                {
                    Data.AddNew();
                    Cards.FocusedColumn = Cards.GetVisibleColumn(0);
                    Cards.ShowEditor();
                    e.Handled = true;
                }
            }
        }

    }   
}

File: VB/WindowsApplication1/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraGrid.Views.Card

Namespace WindowsApplication1
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
			Dim TempXViewsPrinting As DevExpress.XtraGrid.Design.XViewsPrinting = New DevExpress.XtraGrid.Design.XViewsPrinting(gridControl1)
		End Sub

		Private Sub cardView1_KeyDown(ByVal sender As Object, ByVal e As KeyEventArgs) Handles cardView1.KeyDown
			Dim Cards As CardView = TryCast(sender, CardView)
			If e.KeyCode = Keys.Insert AndAlso (Not Cards.IsEditing) Then
				Dim Data As DataView = TryCast(Cards.DataSource, DataView)
				If Data IsNot Nothing Then
					Data.AddNew()
					Cards.FocusedColumn = Cards.GetVisibleColumn(0)
					Cards.ShowEditor()
					e.Handled = True
				End If
			End If
		End Sub

	End Class
End Namespace